DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -148,8 +148,7 @@ public void Configure(IApplicationBuilder app, IMigrationRunner migrationRunner)
         {
             app.UseForwardedHeaders(new ForwardedHeadersOptions
             {
-                ForwardedHeaders = ForwardedHeaders.XForwardedFor |
-                                   ForwardedHeaders.XForwardedProto
+                ForwardedHeaders = ForwardedHeaders.XForwardedFor | ForwardedHeaders.XForwardedProto

```

Comment:
Just to satisfy my curiosity, what does this vertical bar do?

Reply:
After doing a little digging (I initially just copied this from msdn docs to see if it worked) it looks to be an operator that creates a bitwise combination of its (enum) values for the ForwadedHeaders property (https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.httpoverrides.forwardedheaders?view=aspnetcore-3.1).

This is because the property itself is defined with Flags attribute, which seems to basically be a way to specify multiple potential enum combinations. The Hue example and the PhoneService in the docs seems to be a better explanation than I can muster. https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=netcore-3.1

Reply:
Ooh nice, thanks for the research 🙂